guide/src/faq.md

```diff
@@ -180,3 +180,15 @@ done with the `crate` attribute:
 #[pyo3(crate = "reexported::pyo3")]
 struct MyClass;
 ```
+
+## I'm trying to call Python from Rust but I get `STATUS_DLL_NOT_FOUND` or `STATUS_ENTRYPOINT_NOT_FOUND`!
+
+This happens on Windows when linking to the python DLL fails or the wrong one is linked. Your `PATH` variable should contain something like...
+
+```text
+C:\Users\<You>\AppData\Local\Programs\Python\Python310
+```
+
+This directory should have, for Python 3.10, the files `python3.dll` and `python310.dll`. If you have made any changes to your `PATH` or installed things recently, try restarting your shell - any changes will not affect already open shells.
```

Comment:
I also had the converse issue of other programs adding themselves and their internally used DLL to the `PATH` thereby messing things. For example, programs written in Java often include a JRE which can mess up a dedicated installation thoroughly. Or ArcGIS contains a Python installation used for scripts and plug-ins which could place unexpected Python DLL there.

Not sure if this is worth mentioning explicitly here, just thought I'd bring it up for consideration.

Reply:
Yes, I hit this in https://github.com/PyO3/pyo3/issues/2219 😓